django/contrib/gis/db/backends/base/models.py

```diff
@@ -84,12 +89,16 @@ def datum(self):
         return self.srs['datum']
 
     @property
+    def compound(self):
+        return self.wkt.startswith('COMPD_CS')
+
+    @property
     def projected(self):
         "Is this Spatial Reference projected?"
         if gdal.HAS_GDAL:
             return self.srs.projected
         else:
-            return self.wkt.startswith('PROJCS')
+            return self.wkt.startswith('PROJCS') or self.compound and 'PROJCS' in self.wkt
```

Comment:
You might use parenthesis to be explicit about the groupings and improve readability in these longer expressions.